network_manager/models.py

```diff
@@ -25,15 +25,19 @@ class Node(Base):
 
     __tablename__ = "nodes"
 
-    node_id = Column(Integer, primary_key=True, autoincrement=True)
-
-    node_name = Column(Text, unique=True)
-    node_description = Column(Text)
-    location = Column(Text)
-    affiliation = Column(Text)
-    url = Column(Text, unique=True)
-    support_contact_email = Column(Text)
-    deploy_start_date = Column(DateTime)
-    data = Column(Boolean, default=False)
-    compute = Column(Boolean, default=False)
-    administrator = Column(Text)
+    id = Column(Integer, primary_key=True, autoincrement=True)
```

Comment:
column names have been changed to reflect the values in the node info json. 

Also, do not repeat the table name in the column name:

- nodes.node_id <- this is unnecessary repetition
- nodes.id  <- this is better